bin/instructions_makepdf.md

```diff
@@ -0,0 +1,98 @@
+# Instructions for makepdf
+
+## Introduction
+
+`makepdf` generates a pdf version from a manual. Dependent on the parametrisation on the command line, it builds either a given manual or all availale. To create a pdf manual, `asciidoctor-pdf` is used as processor. Compared to the build to html which uses a different software from the same family, the parametrisation of `asciidoctor-pdf` can not use the ui-styles, templates and attributes from the html build. In particular for attribues, this can lead to unresolved "variables" which are printed as defined in the build made. To overcome this issue, an additional software (`yamlparse`) based on bash is used to dynamically create an attrribute list from `site.yml` which is then added as parameter list to the pdf built. Whenever an attribute change is made in `site.yml`, a new pdf build will use these attribues automatically.
+
+##  Manual Usage
+
+To use `makepdf`, call it from the root of the repo. Dependent on the doc repo you use, the list of buildable manuals may differ.
+
+```
+./bin/makepdf -h
+
+Usage: ./bin/makepdf [-c] [-d] [-h] [-m] [-n <admin|user|developer>]
+
+-h ... help
+-c ... clean the build/ directory (contains the pdf)
+-d ... debug mode, print out with what it would work, only with -m
+-m ... build all modules
+-n ... build module <name>, only with -m
+```
+
+##  Docker Usage
+
+[//]: <> (More content and details to be added)
+
+When called from Docker use:
+
+```
+"commands": [
+  "bin/makepdf -m",
+],
+```
+
+## How a pdf is Built
+
+The general idea of `asciidoctor-pdf` is to create a pdf from a given single file and add options to the build command, see the [CLI Options](https://docs.asciidoctor.org/asciidoctor.js/latest/cli/options/) for details. Because the manuals are usually made out of multiple documents, a base document must be created which is then built to pdf. Therefore following method is used: A book template file has to be physically present in the `books/` folder. The following example file shows the basic structure and provides some settings.
```

Comment:
```suggestion
The general idea of `asciidoctor-pdf` is to create a pdf from a given single file and add options to the build command, see the [CLI Options](https://docs.asciidoctor.org/asciidoctor.js/latest/cli/options/) for details. Because manuals are usually made out of multiple documents, a base document must be created which is then built to pdf. Therefore following method is used: A book template file has to be physically present in the `books/` folder. The following example file shows the basic structure and provides some settings.
```